bytes.md

```diff
@@ -0,0 +1,50 @@
+---
+title: BYTES
+toc: true
+---
+
+## Description
+
+The `BYTES` [data type](data-types.html) stores binary strings of variable, unlimited length.
```

Comment:
Good points. For now, I'll just state "variable length". I opened an issue to figure out the exact recommendations (probably post-beta release): issue #159